### PR TITLE
STRF-9124 Create an endpoint for precaching lang helper function

### PR DESCRIPTION
**Pull request by jairo-bc**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -286,9 +286,18 @@ class Paper {
      * 
      * @param {Object} templats 
      */
-    addTemplates(templates) {
+     addTemplates(templates) {
```
Comment (jmwiese) [inline index.js:289]: 🍹 extra space at beginning of line

File: lib/translator/index.js [hunk 2]
```diff
@@ -66,6 +63,53 @@ Translator.create = function (acceptLanguage, allTranslations, logger = console,
...
+    const locale = language.locales[key];
+    const formatter = new MessageFormat(locale)
+
+    try {
+        return formatter.compile(language.translations[key]).toString();
+    } catch (err) {
+        if (err.name === 'SyntaxError') {
+            this.logger.error(`Syntax Error during Formatter function precompilation: ${err.message} for key "${key}"`, err.expected);
+
+            return () => '';
```
Comment (jmwiese) [inline lib/translator/index.js:106]: Is this not a situation where we want to also throw an error? Why do we just want to return empty string here?

File: lib/translator/index.js [hunk 3]
```diff
@@ -104,14 +153,22 @@ Translator.prototype.getLocale = function () {
  * @param {string} [keyFilter]
  * @returns {Object} Language object
  */
-Translator.prototype.getLanguage = function (keyFilter) {
+ Translator.prototype.getLanguage = function (keyFilter) {
```
Comment (jmwiese) [inline lib/translator/index.js:156]: 🍹 Extra space

File: lib/translator/index.js (hunk 2 above)
Comment (jairo-bc) [reply lib/translator/index.js:106]: I'm not sure why it was there, I've just used the same logic as it was at current helper implementation to have compatibility

**Review by jmwiese — APPROVED** [review]

**Pull request merged**